mne/io/nicolet/nicolet.py

```diff
@@ -0,0 +1,212 @@
+                     'range': 1.0, 'unit_mul': 0., 'ch_name': ch_name,
+                     'unit': FIFF.FIFF_UNIT_V,
+                     'coord_frame': FIFF.FIFFV_COORD_HEAD,
+                     'coil_type': coil_type, 'kind': kind, 'loc': np.zeros(12)}
+        info['chs'].append(chan_info)
+    return info, header_info
+
+
+class RawNicolet(_BaseRaw):
+    """Raw object from Nicolet file
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the Nicolet file.
+    montage : str | None | instance of Montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    eog : list or tuple
+        Names of channels or list of indices that should be designated
+        EOG channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    ecg : list or tuple
+        Names of channels or list of indices that should be designated
+        ECG channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    emg : list or tuple
+        Names of channels or list of indices that should be designated
+        EMG channels. If None (default), the channel names beginning with
+        ``EMG`` are used.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    def __init__(self, input_fname, montage=None, eog=None, ecg=None, emg=None,
+                 misc=None, preload=False, verbose=None):
+        input_fname = path.abspath(input_fname)
+        info, header_info = _get_nicolet_info(input_fname, eog, ecg, emg, misc)
+        last_samps = [header_info['num_samples'] - 1]
+        _check_update_montage(info, montage)
+        super(RawNicolet, self).__init__(
+            info, preload, filenames=[input_fname], raw_extras=[header_info],
+            last_samps=last_samps, orig_format='int',
+            verbose=verbose)
+
+    def _read_segment_file(self, data, idx, offset, fi, start, stop, cals,
+                           mult):
+        """Read a chunk of raw data"""
+        if mult is not None:
+            raise NotImplementedError()
```

Comment:
Why do you need this? You should just be able to do what is done with other readers, no?